.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+format = ${cyan}%(path)s${reset}:${yellow_bold}%(row)d${reset}:${green_bold}%(col)d${reset}: ${red_bold}%(code)s${reset} %(text)s
```

Comment:
Kan niet in `pyproject.toml`?

Reply:
No, flake8 does not support `pyproject.toml` and it can't be in `setup.cfg` (see commit message 53f69781da6831b9991b6ceed65c4390be0f51fd)